src/decenc_openjpeg.c

```diff
@@ -369,6 +354,61 @@ dec_jpeg2000(char *injpc, g2int bufsize, g2int *outfld)
     return iret;
 }
 
+/**
+ * This Function decodes a JPEG2000 code stream specified in the
+ * JPEG2000 Part-1 standard (i.e., ISO/IEC 15444-1) using OpenJPEG.
+ *
+ * PROGRAM HISTORY LOG:
+ * - 2002-12-02  Gilbert
+ * - 2016-06-08  Jovic
+ *
+ * @param injpc Input JPEG2000 code stream.
+ * @param bufsize Length (in bytes) of the input JPEG2000 code stream.
+ * @param outfld Output matrix of grayscale image values.
+ *
+ * @return
+ * - 0 Successful decode
+ * - -3 Error decode jpeg2000 code stream.
+ * - -5 decoded image had multiple color components. Only grayscale is expected.
+ *
+ * @note Requires OpenJPEG Version 2.
+ *
+ * @author Alyson Stahl
```

Comment:
List all the authors, adding your name on the end. But that's too minor to hold up the merge, just remeber for next time. ;-)